### PR TITLE
variable to variable references 

### PR DESCRIPTION
**Pull request by marqh**
clarifications on the implementation details for variable to variable references

attempts to address feedback comments from https://github.com/binary-array-ld/net.binary_array_ld.bald/pull/29#issuecomment-817727335

@simonoakesepimorphics your consideration of this PR would be very much appreciated

### CODE REVIEW

**Comment by marqh** [comment]
I am now wondering whether it would help to expand the examples to include a couple of CF specific VtoV refs, such as `coordinates` and bounds` to illustrate the more common usage of this kind of referencing

whilst we can use the `bald:references` objectProperty directly in the standard and in testing, the common use Cases are around the use of referencing in conventions such as CF

I will consider that as additional material within the informative section, but perhaps keep it out of the normative section.  it could be an extension to this PR, or a new PR, if that is useful

**Comment by marqh** [comment]
@simonoakesepimorphics

this isn't in the editor's draft yet, if you want a built copy, please refer to the build instructions for local checkout asciidoc building
https://github.com/opengeospatial/netcdf-ld/blob/master/BUILDING.md

**Comment by marqh** [comment]
CF example added in https://github.com/opengeospatial/netcdf-ld/pull/76/commits/ce7de88a93ffbc1ef696ae9dbe150406c0cc35f3

**Comment by simonoakesepimorphics** [comment]
LGTM

File: standard/clause_6_informative_text.adoc [hunk 1]
```diff
@@ -481,6 +481,17 @@ To identify a predicate as a variable-to-variable reference predicate, that pred
...
 
+Alternatively, it is permissable to provide an rdfs:range statement targeting another resource and for that resource to declare that it is an rdfs:subClassOf bald:Resource:
+
+----
+ <{attribute}> rdfs:range <{AClass}> .
+
+ <{AClass}> rdfs:subClassOf <https://www.opengis.net/def/binary-array-ld/Resource> .
+----
+
+This standard does not support nested derivation, an implementation is not required to search the inheritance tree of a defined Class to see if any of the classes it inherits from define themselves as rdfs:subClassOf bald:Resource.
```
Comment (jyucsiro) [inline standard/clause_6_informative_text.adoc:492]: So only direct subclassOf bald:Resource statements are supported to enable V2V refs. Might be worth stating this in the positive as well?

File: standard/clause_6_informative_text.adoc [hunk 2]
```diff
@@ -481,6 +481,17 @@ To identify a predicate as a variable-to-variable reference predicate, that pred
...
+
+The definition of the owl:objectProperty bald:references uses this rdfs:range, rdfs:subClassOf definition within the bald ontology: https://www.opengis.net/def/binary-array-ld
```
Comment (jyucsiro) [inline standard/clause_6_informative_text.adoc:494]: Should we provide the definition of the bald:references objectProperty inline here to be explicit about what it is?

File: standard/clause_6_informative_text.adoc [hunk 3]
```diff
@@ -533,7 +544,7 @@ For an in file variable reference to be declared, three conditions shall be met.
 *** 'search by absolute path' and 'search by relative path' shall be implemented;
 *** 'search by proximity' is deprecated by CF, and may be implemented or not.
 * Condition two: the attribute name is already interpreted as a URI, defining an entity, external to the file.
-* Condition three: the attribute name entity declares and `<rdfs:range>` of `<bald:Resource>`.
+* Condition three: the attribute name entity declares and `<rdfs:range>` of `<bald:Resource>` (or uses rdf:subClassOf linkage).
```
Comment (jyucsiro) [inline standard/clause_6_informative_text.adoc:547]: Did you mean
```
Condition three: the attribute name entity declares an...
```

rdf:subClassOf linkage -> direct rdf:subClassOf linkage?

File: standard/clause_6_informative_text.adoc [hunk 4]
```diff
@@ -944,6 +955,149 @@ this:location_variable a bald:Array ;
...
+                time:standard_name = "time" ;
+                time:long_name = "Time axis" ;
+                time:calendar = "360_day" ;
+                time:time_origin = "2001-JAN-01 00:00:00" ;
+                time:title = "Time" ;
+// global attributes:
+                :Conventions = "CF-1.5" ;
+----
+
+then the `coordinates` attribute defined on the votemper variable is interpreted as a varaible to variable reference, creating 4 explicit reference statements with blank nodes representing the bald:Reference instances.
```
Comment (jyucsiro) [inline standard/clause_6_informative_text.adoc:1014]: "the votemper variable is interpreted as a varaible to variable" --> 
"the votemper variable is interpreted as a variable to variable"

File: standard/clause_6_informative_text.adoc [hunk 5]
```diff
@@ -944,6 +955,149 @@ this:location_variable a bald:Array ;
...
+These explicit bald:references statements are included within the graph alongside the CFTerms:coordinates statements. 
+
+This can be seen within the interpreted metadata graph for this file
+
+----
+@prefix CFTerms: <http://def.scitools.org.uk/CFTerms/> .
```
Comment (jyucsiro) [inline standard/clause_6_informative_text.adoc:1020]: good

**Review by jyucsiro — COMMENTED** [review]
Some minor changes (typos mainly) but overall reads well and examples match up and are easily understood. Good to go once typos are addressed from my perspective...

File: standard/clause_6_informative_text.adoc (hunk 1 above)
Comment (marqh) [reply standard/clause_6_informative_text.adoc]: l492

File: standard/clause_6_informative_text.adoc (hunk 2 above)
Comment (marqh) [reply standard/clause_6_informative_text.adoc:494]: added

File: standard/clause_6_informative_text.adoc (hunk 3 above)
Comment (marqh) [reply standard/clause_6_informative_text.adoc]: typo fixed

File: standard/clause_6_informative_text.adoc (hunk 4 above)
Comment (marqh) [reply standard/clause_6_informative_text.adoc]: typo fixed

**Pull request merged**